app/assets/stylesheets/move_to_ui_gem.scss

```diff
@@ -29,3 +29,9 @@ hr.dashed {
         padding-left: 26px;
     }
 }
+
+tr {
```

Comment:
I think this should be scoped to just note tr actions, as this might bite us later on.